plugins/statistics.koplugin/main.lua

```diff
@@ -172,6 +195,16 @@ function ReaderStatistics:addToMainMenu(tab_item_table)
                     })
                 end
             },
+            {
+                text = _("Reading Statistics"),
```

Comment:
Looks like it's a break down of stats by different time range. See `getAdvStats` method.